### DISCUSSION TITLE
New bugfix release?

### DISCUSSION THREAD

**Discussion by aried3r** [shrinerb/shrine#582] [General]
Hey!

Since the last release a few minor bug fixes were merged and we were wondering if a new patch release would be possible? We’re currently pointing our Gemfile to GitHub but for easier auditing we’d love to use official releases again.

Is there anything we could do to help make this happen?

Cheers!

PS: https://shrinerb.com/ still points to Discourse for discussions, perhaps it should be changed to GitHub Discussions?

**Comment by benkoshy** [comment]
> PS: https://shrinerb.com/ still points to Discourse for discussions, perhaps it should be changed to GitHub Discussions?

Fixed with PR: [#583](https://github.com/shrinerb/shrine/pull/583)
